README.md

```diff
@@ -5,11 +5,16 @@ It does so by providing on-screen translation ability, so that the community can
 ![Screenshot](docs/screenshot.png)
 
 ### Requirements
-- jQuery
-- The [`gp-translation-extended-api`](https://github.com/automattic/gp-translation-extended-api) plugin for GlotPress
+- The [GlotPress as a WordPress plugin](https://github.com/GlotPress/GlotPress-WP)
+- The [`gp-translation-extended-api`](https://github.com/david-binda/gp-extended-api-plugins) plugin for GlotPress as a WordPress plugin
 
 ### Build
-To create the `community-translator.js` and `community-translator.css` files which should be loaded in the translatable site, run `make`.
+To create the `community-translator.js` and `community-translator.css` files which should be loaded in the translatable site.
+
+* Make sure you've got uglifyjs installed in system. If not, run `sudo npm install -g uglify-js`
+* run `npm install`
+* run `make`.
```

Comment:
That's not great on our part :/  I'd much prefer to add uglify-js as a dev-dependency, but I'll merge this PR and take it back out in that one.